packages/terra-text/src/terra-dev-site/doc/example/TextWeights.jsx

```diff
@@ -3,6 +3,12 @@ import Text from 'terra-text';
 
 const TextWeights = () => (
   <div>
+    <p>
+      The `weight` prop can be used to specify the weight or boldness of the `Text` component. Heading components default to a `font-weight` of `700`. The `weight` prop accepts one of the following numbers:
+      <br />
+      <br />
+      `200`, `400`, `700`
+    </p>
```

Comment:
IMO these description can be dropped since it's describe (or should be described) in the props table

Reply:
Updated here https://github.com/cerner/terra-core/pull/2886/commits/6ec1a2ee4e63a17068af0be5b572b0f7f6ac901e.